packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayoutWithoutModalSwitcher.tsx

```diff
@@ -35,6 +35,7 @@ const Content = styled.div<{ $elevation: Elevation; $verticalCenter?: boolean }>
     align-items: center;
     overflow-y: auto;
     height: 100%;
+    max-width: ${MAX_WELCOME_LAYOUT_WIDTH}px;
```

Comment:
We should use `MAX_CONTENT_WIDTH` here, because it's shouldn't be "welcome screen" anymore

Also would be nice to create a common layout container like `ContentWrapper` in `SuiteLayout.tsx` for all usages.